doc/xplat_cli_migration.md

```diff
@@ -0,0 +1,223 @@
+# Migrating from Azure XPlat CLI to Azure CLI 2.0
+
+With the introduction of our Azure CLI 2.0 preview, customers now have a
```

Comment:
>Azure CLI 2.0 preview [](start = 29, length = 21)

sometimes we capitalize preview, we should probably unify that across the docs.  I'm guessing lowercase?